### ISSUE TITLE
JS Callback on Preview Render in WYSIWIG

### ISSUE THREAD

**Issue by paulkarpenko** [wp-shortcake/shortcake#326]
Has there been any discussion of allowing a callback to be registered on .done() in fetchShortcode? this would be useful for allowing JS DOM work to be done once a shortcode component is rendred in the WYSIWIG. And does a workaround currently exist for this without having to edit the plugin's JS?

**Comment by paulkarpenko** [comment]
It looks like https://github.com/fusioneng/Shortcake/issues/85 also addresses this need. Has any additional work been done here?

**Comment by danielbachhuber** [comment]
> Has any additional work been done here?

Nope. I'm not sure what our official opinion is in this regard at the moment.
